istioctl/pkg/writer/envoy/configdump/listener.go

```diff
@@ -534,6 +534,9 @@ func describeMatch(match *route.RouteMatch) string {
 	if match.GetPrefix() != "" {
 		conds = append(conds, fmt.Sprintf("%s*", match.GetPrefix()))
 	}
+	if match.GetPathSeparatedPrefix() != "" {
+		conds = append(conds, fmt.Sprintf("%s or %s?* or %s/*", match.GetPathSeparatedPrefix(), match.GetPathSeparatedPrefix(), match.GetPathSeparatedPrefix()))
```

Comment:
As you mentioned, this might be a long format. I think the second option is better; however, could we use a more user-facing term instead of the envoy field name?

Reply:
Naming is hard... Do you mean something like this:
```
NAME        VHOST NAME                 DOMAINS                 MATCH                   VIRTUAL SERVICE
http.80     httpbin.example.com:80     httpbin.example.com     path-separated /get     http-0-istio-autogenerated-k8s-gateway.default
            backend                    *                       /stats/prometheus*           
            backend                    *                       /healthz/ready* 
```

Or maybe express it in a regex, but is it clear? 
```
NAME        VHOST NAME                 DOMAINS                 MATCH                                   VIRTUAL SERVICE
http.80     httpbin.example.com:80     httpbin.example.com     \/get(\?(?:.*&)?(.*)=[^&]*|\/(.*)|)     http-0-istio-autogenerated-k8s-gateway.default
            backend                    *                       /stats/prometheus*           
            backend                    *                       /healthz/ready* 
```

Reply:
`PathMatchType` is a string of value `Exact|PathPrefix|RegularExpression`, couldn't we just do something like
```
NAME        VHOST NAME                 DOMAINS                 MATCH                    VIRTUAL SERVICE
http.80     httpbin.example.com:80     httpbin.example.com     PathPrefix /get, ...     http-0-istio-autogenerated-k8s-gateway.default
            backend                    *                       /stats/prometheus*           
            backend                    *                       /healthz/ready* 
```

Reply:
I like it, thanks!

Reply:
or maybe along the same lines,`PathPrefix:/get` instead of a space?

Reply:
We can choose either format, as I think there is no similar format in istioctl currently, so no worries for consistency.

> or maybe along the same lines,PathPrefix:/get instead of a space?

I personally prefer this one with better readability.

Reply:
Done